Templates/DefaultProject/template.json

```diff
@@ -142,6 +142,10 @@
             "file": "Gem/gem.json",
             "isTemplated": true
         },
+        {
+            "file": "Levels/Default/DefaultLevel.prefab",
```

Comment:
I'm kinda surprised it made the foldername different than the prefab name, but fine with it if it works.